api/app/views/spree/api/variants/small.v1.rabl

```diff
@@ -14,4 +14,4 @@ child :option_values => :option_values do
   attributes *option_value_attributes
 end
 
-child(:images => :images) { extends "spree/api/images/show" }
+child(:display_images => :images) { extends "spree/api/images/show" }
```

Comment:
> my understanding was that we were going to have rule-based images at the product-level in the api response

How would this work for the varuant search route?


> In my opinion the performance impact is very small.

My intuition is otherwise, have you tested this?